oci/spec_opts_windows.go

```diff
@@ -100,3 +100,32 @@ func escapeAndCombineArgs(args []string) string {
 	}
 	return strings.Join(escaped, " ")
 }
+
+// WithWindowsDevice adds a device exposed to a Windows (WCOW or LCOW) Container
+func WithWindowsDevice(IDType, ID string) SpecOpts {
+	return func(_ context.Context, _ Client, _ *containers.Container, s *Spec) error {
+		if len(IDType) == 0 {
```

Comment:
I cant say I've ever seen it this way :). Is this preferable to `IDType == ""` for some golang reason I don't know?

Reply:
No, that's probably my background in C/C++ making me adverse to string comparisons in principle. I did wonder which one was idiomatic in Go but forgot to come back and check it.